website/client-old/js/services/taskServices.js

```diff
@@ -72,10 +72,16 @@ angular.module('habitrpg')
     };
 
     function updateTask (taskId, taskDetails) {
+      var taskDetailsToSend = _.omit(
+        taskDetails,
+        ['challenge', 'completed', 'createdAt', 'group', 'history', 'id',
```

Comment:
Though perhaps it wouldn't be obvious why we're keeping just those properties off of the update call.